bin/alluxio-start.sh

```diff
@@ -137,6 +142,17 @@ stop() {
   ${BIN}/alluxio-stop.sh $1
 }
 
+start_logservers() {
```

Comment:
why do we need multiple log servers? Could we use just one?

Reply:
We can do with one log server, but the log server needs to have multiple server threads, each of which listens on a different port. Each port corresponds to a type of Alluxio server, e.g. masters, workers, etc.

Reply:
It looks like `LoggingEvent` has a `getLoggerName()` method - could we use this instead of the port to determine what folder to create a log file inside of? It would be cleaner if we could use just one port.

Reply:
I've come up with a solution that requires some code refactoring. Will push shortly.

Reply:
Done.